htdocs/libraries/icms/view/theme/Object.php

```diff
@@ -215,12 +215,15 @@ public function xoInit($options = array()) {
 			} elseif (substr($name, 0, 6) == 'footer') {
 				$values = $value;
 							if ($icmsConfigMetaFooter['use_google_analytics'] == TRUE && isset($icmsConfigMetaFooter['google_analytics']) && $icmsConfigMetaFooter['google_analytics'] != '') {
-					$values = $value . '<script type="text/javascript">
-                      var _gaq = _gaq || [];  _gaq.push(["_setAccount", "UA-' . $icmsConfigMetaFooter['google_analytics'] . '"]);  _gaq.push(["_trackPageview"]);
-                      (function() {var ga = document.createElement("script"); ga.type = "text/javascript"; ga.async = true;
-                      ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";
-                      var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(ga, s);})();
-                    </script>';
+					$values = $value . "<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','
```

Comment:
@fiammybe part of this line appears to be missing. Looking at the code snippet for our properties and mine, it also needs '//www.google-analytics.com/analytics.js','ga');